info-compiler/src/test/java/com/google/sps/NewsContentProcessorTest.java

```diff
@@ -64,7 +65,7 @@ public void process_longContent() {
     NewsArticle processedNewsArticle = NewsContentProcessor.process(newsArticle);
     NewsArticle expectedArticle = new NewsArticle(newsArticle);
     expectedArticle.setAbbreviatedContent(MAX_CONTENT);
-    Assert.assertEquals(expectedArticle, processedNewsArticle);
+    assertThat(expectedArticle).isEqualTo(processedNewsArticle);
```

Comment:
Reverse tested object/expected ordering
```suggestion
    assertThat(processedNewsArticle).isEqualTo(expectedArticle);
```

Reply:
Yeah sorry I'm currently working these!

Reply:
Oop sorry will wait till you add me back to the review list before my next review :)

Reply:
Ah I should've pinged you or left a message. My bad!
I will get these revisions in right away

Reply:
haha take your time, i was overeager